lib/action_policy/policy/core.rb

```diff
@@ -20,15 +20,22 @@ class UnknownRule < Error
 
     attr_reader :policy, :rule, :message
 
-    def initialize(policy, rule, message = nil)
+    def initialize(policy, rule)
       @policy = policy.class
       @rule = rule
-      @message = message ||
-        "Couldn't find rule '#{@rule}' for #{@policy}" \
+      @message = "Couldn't find rule '#{@rule}' for #{@policy}" \
         "#{suggest(@rule, @policy.instance_methods - Object.instance_methods)}"
     end
   end
 
+  class NonPredicateRule < UnknownRule
+    def initialize(policy, rule)
+      @policy = policy.class
+      @rule = rule
+      @message = "The rule '#{@rule}' of '#{@policy}' must ends with ? (question mark)\nDid you mean? #{@rule}?"
```

Comment:
Let's use the same suggest as above (and `did_you_mean` will do the trick for us).

We can extract `suggest(@rule, @policy.instance_methods - Object.instance_methods)` into a method (say, `suggestion(rule)`) and use it here.

Reply:
method that we want to reference is not exists, e.g. I have `default_rule :manage?` and want suggestion for `create`, but `create?` method is not exits

Reply:
it works for ```
def bad?
end```
but not for ```
  def update?
  end
  alias_rule :bad?, to: :update?
```

I sure that SuggestMessage is not need here, because that's about enforce_predicate_rules_naming not about other methods